simulators/active_surface.py

```diff
@@ -7,6 +7,9 @@
 
 class Driver(object):
 
+    standby_delay_step = 0.004096
+
+    # Resolution denominator, i.e.: 0 -> 1/1, 1 -> 1/2, ..., 3 -> 1/8, ...
```

Comment:
If you think this comment is important and it should be added to the doc, then write it in this way, in order for Sphinx to put the comment in the doc: `#: Resolution...`.  If you do not think it should go to the doc, leave it as it is now.  Apply the same role for all other comments in this file.